microservices/services/tracking-service/src/main/java/org/xcolab/service/tracking/service/TrackedVisitor2UserService.java

```diff
@@ -20,7 +20,10 @@ public TrackedVisitor2UserService(TrackedVisitor2UserDao trackedVisitor2UserDao)
         this.trackedVisitor2UserDao = trackedVisitor2UserDao;
     }
 
-    public TrackedVisitor2User getOrCreate(long memberId) {
+    public TrackedVisitor2User getOrCreate(Long memberId) {
+        if (memberId == null) {
+            return create();
```

Comment:
This doesn't make sense to me. Why does passing a `memberId == null` result in a new `TrackedVisitor2User` being created?

Reply:
If we pass a `null` memberId, that means we don't know the user. In this case, rather than looking for an existing visitor, we create a new one without setting a memberId (an unknown visitor). You're right that this is a bit confusing, I tried making it a bit clearer by renaming the method. Is this more understandable now?